src/AlgebraicGeometry/Schemes/ProjectiveAlgebraicSet/Objects/Attributes.jl

```diff
@@ -37,12 +37,11 @@ fat_ideal(X::AbsProjectiveAlgebraicSet) = defining_ideal(fat_scheme(X))
 
 
 base_ring(X::AbsProjectiveAlgebraicSet) = base_ring(fat_scheme(X))
+ambient_coordinate_ring(X::AbsProjectiveAlgebraicSet) = ambient_coordinate_ring(fat_scheme(X))
 
 dim(X::AbsProjectiveAlgebraicSet) = dim(fat_scheme(X))
 
 codim(X::AbsProjectiveAlgebraicSet) = codim(fat_scheme(X))
 
-homogeneous_coordinate_ring(X::AbsProjectiveAlgebraicSet) = homogeneous_coordinate_ring(fat_scheme(X))
```

Comment:
@HechtiDerLachs As you see this line was wrong and therefore deleted.  (My bad, overlooked it in a code review)


In 
`.julia/dev/Oscar/experimental/DoubleAndHyperComplexes/test/ext.jl:357` the canonical bundle of a (complicated) variety `X`. Is computed via a resolution of its `homogeneous_coordinate_ring`. 

Now with the correct homogeneous coordinate ring this triggers the computation of a radical which takes ages (ofcourse). 

How would you like to proceed? Can we avoid the radical? Is the ideal you give a radical ideal anyways? (then we could tell Oscar and avoid the computaton).

Reply:
> Now with the correct homogeneous coordinate ring this triggers the computation of a radical which takes ages (ofcourse).

Does this also effect other examples already in the docu? For example, without the change, I get:
```julia-repl
julia> X = bordiga();

julia> @time X = rational_d9_pi7();
  0.031707 seconds (272.94 k allocations: 13.039 MiB)
```
What do you get now?

Reply:
`homogeneous_coordinate_ring(P::AbsProjectiveScheme) = homogeneous_coordinate_ring(underlying_scheme(P))
`
in `src/AlgebraicGeometry/Schemes/ProjectiveSchemes/Objects/Attributes.jl` 
and `underlying_scheme` computes the radical.

Reply:
this is called in 
```
function canonical_bundle(X::AbsProjectiveVariety)
  Pn = ambient_coordinate_ring(X)
  A = homogeneous_coordinate_ring(X)
  n = ngens(Pn)-1
  c = codim(X)
  FA, _ = free_resolution(SimpleFreeResolution, A)
......
```

Reply:
O.K. I told Oscar that `I` is a radical ideal already. Please confirm @wdecker 

In the `adjunction_process` there is also an `Y = variety(I, check = false, is_radical = false)` which may lead to radical computations. Is this intended @wdecker ? (just double checking)

Reply:
> > Can we avoid the radical? Is the ideal you give a radical ideal anyways?
> 
> I suppose so. But the example is due to @wdecker , so he should know, in fact.

Confirmed!

Reply:
> Is this intended @wdecker ? (just double checking)

No, I was not really aware of the consequences (expensive testing). Please set `is_radical = true`.

Reply:
There is another example in `.julia/dev/Oscar/src/AlgebraicGeometry/Surfaces/SurfacesP4.jl` which needs to be changed now:

```julia-repl
function surface(n::String)
  n = joinpath(oscardir, "data", "Surfaces", "$n" * ".mrdi")
  I = load(n)
  S = base_ring(I)
  R = grade(S)[1]
  #return proj(R, ideal(R, map(R, gens(I))))
  return variety(ideal(R, map(R, gens(I))), check = false, is_radical = false,)
end
```

Reply:
Okay,  but let us keep this for another PR. 

It is explained to the user in the `variety(I::MPolyIdeal; check=true) -> AffineVariety` docstring
https://docs.oscar-system.org/stable/AlgebraicGeometry/AlgebraicVarieties/AffineVariety/#variety-Tuple{MPolyIdeal}

But perhaps we should turn it into a red **warning** box.
```
By our convention, varieties are absolutely irreducible. Hence we check that the radical of II is prime and stays prime when viewed over the algebraic closure. This is an expensive check that can be disabled.
```